.gitignore

```
@@ -0,0 +1,68 @@
+# Compiled Object files
+*.slo
+*.lo
+*.o
+*.obj
+
+# Precompiled Headers
+*.gch
+*.pch
+*.ipch
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+*.dll
+
+# Fortran module files
+*.mod
+
+# Compiled Static libraries
+*.lai
+*.la
+*.a
+*.lib
+
+# Executables
+*.exe
+*.out
+*.app
+
+# vim tags
+tags
+.tags
+.*.swp
+
+# Editors
+.vscode
+
+# build-in-source directory
+build*
```

Simple `build*` ignores too much of everything, like if we want to add script `build-all.py` or whatever.
```suggestion
build/*
```
I didn't want do request any changes, because it's actually not important and we can live we any kind of `.gitignore` file except of `*`.